src/main/java/application/rest/v1/ResourcesEndpoint.java

```diff
@@ -0,0 +1,1798 @@
+                response = processResources(client, getItemsAsList(client, volumeSnapshotClassO), VOLUME_SNAPSHOT_CLASS_KIND, VOLUME_SNAPSHOT_CLASS_PROPERTY_NAME, getAPIVersion(VOLUME_SNAPSHOT_GROUP, V1ALPHA1_VERSION)); 
+            }
+
+            if (kind.equalsIgnoreCase(VOLUME_SNAPSHOT_CONTENT_KIND)) {                          
+                final Object volumeSnapshotContentO = coa.listClusterCustomObject(VOLUME_SNAPSHOT_GROUP, V1ALPHA1_VERSION, VOLUME_SNAPSHOT_CONTENT_PLURAL, null, null, null, null);                                                                  
+                response = processResources(client, getItemsAsList(client, volumeSnapshotContentO), VOLUME_SNAPSHOT_CONTENT_KIND, VOLUME_SNAPSHOT_CONTENT_PROPERTY_NAME, getAPIVersion(VOLUME_SNAPSHOT_GROUP, V1ALPHA1_VERSION)); 
+            }
+
+            if (kind.equalsIgnoreCase(WAS_ND_CELL_KIND)) {
+                final Object wasNDCellO;
+                if (namespace == null || namespace.length() == 0)
+                    wasNDCellO = coa.listClusterCustomObject(KAPPNAV_GROUP, APP_VERSION, WAS_ND_CELL_PLURAL, null, null, null, null);                         
+                else
+                    wasNDCellO = coa.listNamespacedCustomObject(KAPPNAV_GROUP, APP_VERSION, namespace, WAS_ND_CELL_PLURAL, null, null, null, null);                         
+                response = processResources(client, getItemsAsList(client, wasNDCellO), WAS_ND_CELL_KIND, WAS_ND_CELL_PROPERTY_NAME, getAPIVersion(KAPPNAV_GROUP, APP_VERSION));               
+            }
+
+            if (kind.equalsIgnoreCase(WAS_TRADITIONAL_APP_KIND)) {
+                final Object twasAppO;
+                if (namespace == null || namespace.length() == 0)
+                    twasAppO = coa.listClusterCustomObject(KAPPNAV_GROUP, APP_VERSION, WAS_TRADITIONAL_APP_PLURAL, null, null, null, null);                         
+                else
+                    twasAppO = coa.listNamespacedCustomObject(KAPPNAV_GROUP, APP_VERSION, namespace, WAS_TRADITIONAL_APP_PLURAL, null, null, null, null);                         
+                response = processResources(client, getItemsAsList(client, twasAppO), WAS_TRADITIONAL_APP_KIND, WAS_TRADITIONAL_APP_PROPERTY_NAME, getAPIVersion(KAPPNAV_GROUP, APP_VERSION));               
+            }
+           
+            return response;
+
+        } catch (IOException | ApiException e) {
+            if (Logger.isErrorEnabled()) {
+                Logger.log(className, "getResources", Logger.LogType.ERROR, "Caught Exception returning status: " + getResponseCode(e) + " " + e.toString());
+            }
+            return Response.status(getResponseCode(e)).entity(getStatusMessageAsJSON(e)).build();
+        }   
+    }
+   
+    private Response processResources(final ApiClient client, final List<JsonObject> objects, final String kind, final String propName, final String apiVersion) {
```

Comment:
Returning action and section lists is not a requirement for this API.  The use case is to return resources for the "Add Components" dialog in the UI.  That does not expect actions/sections.  This should be removed.

Reply:
ok

Reply:
ok, I will remove them